artifactory/commands/buildinfo/publish.go

```diff
@@ -91,12 +96,50 @@ func (bpc *BuildPublishCommand) Run() error {
 	if err != nil {
 		return err
 	}
+
+	buildLink, err := bpc.constructBuildInfoUiUrl(servicesManager, buildInfo.Started)
```

Comment:
I suggest returning before this construction if on `dryRun`. The preview build was already logged to the terminal by the client, and a link was not printed prior to these changes.

Reply:
Good point @RobiNino. I'll add this change to v2 as well.